system/Cache/ResponseCache.php

```diff
@@ -102,7 +102,12 @@ public function make(CLIRequest|IncomingRequest $request, ResponseInterface $res
 
         return $this->cache->save(
             $this->generateCacheKey($request),
-            serialize(['headers' => $headers, 'output' => $response->getBody()]),
+            serialize([
+                'headers'      => $headers,
+                'output'       => $response->getBody(),
+                'statuscode'   => $response->getStatusCode(),
+                'reasonphrase' => $response->getReasonPhrase(),
```

Comment:
```suggestion
                'status'  => $response->getStatusCode(),
                'reason' => $response->getReasonPhrase(),
```
Let's make it short.